### DISCUSSION TITLE
Error: "Cell is Empty" when Savings BERTopic Model

### DISCUSSION THREAD

**Discussion by dherysetiaone** [MaartenGr/BERTopic#1619] [Q&A] [answered]
Hi!

I wanted to ask an error when savings BERTopic model
<img width="604" alt="Untitled" src="https://github.com/MaartenGr/BERTopic/assets/139687552/03dc8aaa-751e-4dd6-9598-a8e4139d771c">

It is said that the cell is empty
<img width="571" alt="image" src="https://github.com/MaartenGr/BERTopic/assets/139687552/c830c01d-2d2b-4151-a23b-a4fec8854b3b">

Why is this happen and how can i fix it?
Any kind of help would be appreciated

**Comment by MaartenGr** [comment, marked as answer]
This might be an issue with Python 3.7 according to [this issue](https://github.com/feast-dev/feast/issues/2615). Could you try it with python 3.8 instead?

**Reply by dherysetiaone** [reply]
It works. Thank you.
